docs/production.md

```diff
@@ -68,10 +67,11 @@ This use case assumes that you are on a box that has SSH access to the target bo
 3. Enter the repository - `cd forklift`
 4. [Create an inventory file](https://docs.ansible.com/ansible/latest/intro_inventory.html) with your hosts in the `inventories` subfolder. You can also pass a different inventory to `ansible-playbook` using `--inventory` or `-i`.
 5. Change the variable remote_user in ./ansible.cfg to the appropriate value (the user that will log into the remote machine)
+6. Determine the compatible versions of Foreman and Katello you want to install based on the Katello install instructions at https://github.com/theforeman/forklift/blob/master/vagrant/config/versions.yaml. Replace the 3.16 in the URL with whatever version of Katello you want to install.
```

Comment:
Reading this, I don't know what `3.16` refers to since I don't see any URL.

Reply:
@ekohl the old URL was https://www.theforeman.org/plugins/katello/3.16/installation/index.html but that got changed in a previous commit. So, yeah, the reference to 3.16 needs to be removed.

Reply:
fixed here: https://github.com/theforeman/forklift/pull/1185/commits/2a2728132523822d3e0d78f084ef3b1e999a21de